src/server/index.js

```diff
@@ -143,6 +143,30 @@ app.post('/api/register', function(req, res) {
   }
 });
 
+function responseRequiredFieldsMissing(res) {
+  res.status(HTTP_400).json(responseMessage.REQUIREDFIELDS_MISSING);
+}
+
+function responseCreateBusinessSuccess(res) {
+  res.status(HTTP_201).json(responseMessage.CREATE_BUSINESS_SUCCESS);
+}
+app.post('/api/businesses', function(req, res) {
+  if (req.headers['content-type'] !== 'application/json') {
+    (responseContentTypeError(res));
+  } else if (!req.body.name || !req.body.description ||
```

Comment:
i'd create a `validateRequestBody` or `validateBusinessObject` function for this